graal-tools/src/main/java/io/cloudstate/graaltools/Substitutions.java

```diff
@@ -42,43 +42,6 @@
   private static long ADDRESS_FIELD_OFFSET;
 }
 
-@TargetClass(
-    className =
-        "io.grpc.netty.shaded.io.netty.util.internal.shaded.org.jctools.util.UnsafeRefArrayAccess")
-final
-class Target_io_grpc_netty_shaded_io_netty_util_internal_shaded_org_jctools_util_UnsafeRefArrayAccess {
-  @Alias
-  @RecomputeFieldValue(kind = RecomputeFieldValue.Kind.ArrayIndexShift, declClass = Object[].class)
-  public static int REF_ELEMENT_SHIFT;
-}
```

Comment:
Btw, these were from `proxy/cassandra/src/main/java/io/cloudstate/internal/svm/Substitutions.java`.  Does that still build without these?

Reply:
@dwijnand GraalVM 20.0.0 adds these Substitutions itself.